independent-projects/tools/devtools-common/src/test/resources/dir/some-file.template

```diff
@@ -0,0 +1,3 @@
+Humble smack Shirelings tears baby usurper mouthful. Embrace fresh burned oil Wargs regurgitation unheard-of. Brandybucks puppet vines boats familiar prizewinners tunnel leaving riverbank welcome threw 60? Durin's certain washing crawled children? Pouf Dwarvish elected fruity writing. Laughing reserves hinges Barad-dur either troublemakers saw pleased thereof set weren't penny. Hear my voice. Come back to the light. Slight keyhole strong Shirkers 3434 promised?
```

Comment:
i need to know - what is that generated from ? :)

Reply:
This is a Lorem Ipsum but inspired from GoT if I remember right :)